packages/core/src/logger.ts

```
@@ -1,271 +1,66 @@
-            icon: string;
-            groupTitle: string;
-        }
-    ) {
-        const { fg, bg, icon, groupTitle } = options;
-
-        if (strings.length > 1) {
-            if (this.isNode) {
-                const c = this.#getColor(fg, bg);
-                console.group(c, (this.useIcons ? icon : "") + groupTitle);
+                const message = messageParts.join(" ");
+                return method.apply(this, [arg1, message]);
```

Biome (1.9.4)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)